### PR TITLE
Feat: activity tab

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- applications and moderation activity - Vibe dashboard

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/moderation/src/components/common/page-header.tsx [hunk 1]
```diff
@@ -27,9 +27,12 @@ export const PageHeader: React.FC<PropsWithChildren<IPageHeaderProps>> = props =
 
       <Box customStyle="p-4">{children}</Box>
 
-      <Box customStyle="flex space-x-6 items-center justify-end p-4 my-2">
-        <PageButtons {...props} />
-      </Box>
+      {/* show buttons only when the labels are specified */}
+      {(props.cancelButtonLabel || props.cancelButtonLabel) && (
```
Comment (didd) [inline ui/apps/moderation/src/components/common/page-header.tsx]: May be you meant 
```suggestion
      {props.cancelButtonLabe && (
```

File: ui/apps/moderation/src/components/dashboard/assign-admin.tsx [hunk 2]
```diff
@@ -8,14 +8,14 @@ import Text from '@akashaorg/design-system-core/lib/components/Text';
 import ModeratorListItem from './moderator-list-item';
 import { IPageHeaderProps, PageHeader } from '../common';
 
-export interface IAssignAdminProps extends IPageHeaderProps {
+export type IAssignAdminProps = IPageHeaderProps & {
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/assign-admin.tsx]: The initial `I` wouldn't make sense when used with `type`

File: ui/apps/moderation/src/components/common/page-header.tsx (hunk 1 above)
Comment (josenriagu) [reply ui/apps/moderation/src/components/common/page-header.tsx]: lol, nah, confirmButtonLabel instead. Nice catch

File: ui/apps/moderation/src/components/dashboard/assign-admin.tsx (hunk 2 above)
Comment (josenriagu) [reply ui/apps/moderation/src/components/dashboard/assign-admin.tsx]: gotcha. I will check all those again

File: ui/apps/moderation/src/components/dashboard/guest/index.tsx [hunk 3]
```diff
@@ -6,9 +6,9 @@ import { NavigateToParams } from '@akashaorg/typings/ui';
 
 const { ModerationIntroCard } = DS;
 
-export interface IGuestProps {
+export type IGuestProps = {
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/guest/index.tsx]: check

File: ui/apps/moderation/src/components/dashboard/tabs/applications/application-detail.tsx [hunk 4]
```diff
@@ -5,9 +5,9 @@ import Divider from '@akashaorg/design-system-core/lib/components/Divider';
 import Pill from '@akashaorg/design-system-core/lib/components/Pill';
 import Text from '@akashaorg/design-system-core/lib/components/Text';
 
-import { IPageHeaderProps, PageHeader } from '../../../common';
+import { PageHeaderProps, PageHeader } from '../../../common';
 
-export type IApplicationDetailProps = IPageHeaderProps & {
+export type IApplicationDetailProps = PageHeaderProps & {
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/tabs/applications/application-detail.tsx]: check

File: ui/apps/moderation/src/components/dashboard/tabs/general/categories.tsx [hunk 5]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
-import { PageHeader, IPageHeaderProps, CategoryPills, ICategoryPillsProps } from '../../../common';
+import { PageHeader, PageHeaderProps, CategoryPills, CategoryPillsProps } from '../../../common';
 
-export type IEditCategoriesProps = IPageHeaderProps & ICategoryPillsProps;
+export type IEditCategoriesProps = PageHeaderProps & CategoryPillsProps;
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/tabs/general/categories.tsx]: check

File: ui/apps/moderation/src/components/dashboard/tabs/general/contact-info.tsx [hunk 6]
```diff
@@ -6,9 +6,9 @@ import Text from '@akashaorg/design-system-core/lib/components/Text';
 
 import TextField from '@akashaorg/design-system-core/lib/components/TextField';
 
-import { IPageHeaderProps, PageHeader } from '../../../common';
+import { PageHeaderProps, PageHeader } from '../../../common';
 
-export type IEditContactInfoProps = IPageHeaderProps & {
+export type IEditContactInfoProps = PageHeaderProps & {
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/tabs/general/contact-info.tsx]: check

File: ui/apps/moderation/src/components/dashboard/tabs/general/max-applicants.tsx [hunk 7]
```diff
@@ -5,9 +5,9 @@ import Text from '@akashaorg/design-system-core/lib/components/Text';
 
 import TextField from '@akashaorg/design-system-core/lib/components/TextField';
 
-import { IPageHeaderProps, PageHeader } from '../../../common';
+import { PageHeaderProps, PageHeader } from '../../../common';
 
-export type IEditMaxApplicantsProps = IPageHeaderProps & {
+export type IEditMaxApplicantsProps = PageHeaderProps & {
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/tabs/general/max-applicants.tsx]: check

File: ui/apps/moderation/src/components/dashboard/tabs/general/resign-role.tsx [hunk 8]
```diff
@@ -4,9 +4,9 @@ import Box from '@akashaorg/design-system-core/lib/components/Box';
 import Text from '@akashaorg/design-system-core/lib/components/Text';
 import TextField from '@akashaorg/design-system-core/lib/components/TextField';
 
-import { IPageHeaderProps, PageHeader } from '../../../common';
+import { PageHeaderProps, PageHeader } from '../../../common';
 
-export type IResignRoleProps = IPageHeaderProps & {
+export type IResignRoleProps = PageHeaderProps & {
```
Comment (didd) [inline ui/apps/moderation/src/components/dashboard/tabs/general/resign-role.tsx]: check

**Review by didd — APPROVED** [review]

**Review by didd — APPROVED** [review]

**Pull request merged**